Configurations/unix-Makefile.tmpl

```diff
@@ -1032,10 +1032,10 @@ $obj$objext: $deps
 EOF
           if (defined $makedepprog  && $makedepprog =~ /\/makedepend/) {
               $recipe .= <<"EOF";
-	-\$(MAKEDEPEND) -f- -o"|\$\@" -- $incs $cmdflags -- $srcs \\
-	    >$obj$depext.tmp 2>/dev/null
-	-\$(PERL) -i -pe 's/^.*\\|//; s/ \\/(\\\\.|[^ ])*//; \$\$_ = undef if (/: *\$\$/ || /^(#.*| *)\$\$/); \$\$_.="\\n" unless !defined(\$\$_) or /\\R\$\$/g;' $obj$depext.tmp
-	\@if cmp $obj$depext.tmp $obj$depext > /dev/null 2> /dev/null; then \\
+	\$(MAKEDEPEND) -f- -o"|\$\@" -- $incs $cmdflags -- $srcs 2>/dev/null \\
+	    | \$(PERL) \$(SRCDIR)/util/postprocess-makedepend.pl \\
+	               'makedepend' > $obj$depext.tmp
+	if cmp $obj$depext.tmp $obj$depext > /dev/null 2> /dev/null; then \\
```

Comment:
Why is `@` omitted?

Reply:
Because I find it confusing to see an output to `foo/blah.d.tmp` and then not see what happens with it.

Reply:
Since it, comparison, is not needed in other cases, maybe one can omit it even here? Or what am I missing? [I can't think quite straight yet.]

Reply:
You're missing nothing, that's actually a good point...